### PR TITLE
[SPEEDMERGE THIS ALREADY] Saying "owo" or "uwu" now gives you brain damage

### PR DESCRIPTION
**Pull request by SireTurret**
[Guidelines]: # 

## Changelog
:cl: Turret
add: Furries, mexicans and brazilians eternally BTFO.
/:cl:

## About The Pull Request

This is pretty self-explanatory.
## Why It's Good For The Game

GOD BLESS AMERICA

### CODE REVIEW

**Comment by notoriousBIGGIOS** [comment]
+1

**Review by notoriousBIGGIOS — COMMENTED** [review]
seemsgood

File: hippiestation/code/modules/mob/living/say.dm [hunk 1]
```diff
@@ -23,6 +23,8 @@
 		return
 
 	if(findtext(message, "rouge"))	//You're an idiot and should feel as such
+	if(findtext(message, "owo"))	//You're a furfag and should die
```
Comment (ghost) [inline hippiestation/code/modules/mob/living/say.dm]: ```if(findtext(message, "rouge") || findtext(message, "owo") || findtext(message, "uwu"))```

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — DISMISSED** [review]

**Comment by TrickyDickRichard** [comment]
yes

**Comment by SireTurret** [comment]
speedmerge

**Comment by Majick1** [comment]
+1

**Comment by mikolaj007** [comment]
-1 UwU

File: hippiestation/code/modules/mob/living/say.dm [hunk 2]
```diff
@@ -22,7 +22,7 @@
 	if (!.)
 		return
 
-	if(findtext(message, "rouge"))	//You're an idiot and should feel as such
+	if(findtext(message, "rouge") || findtext(message, "owo") || findtext(message, "uwu")) //thanks steamp0rt
```
Comment (GuyonBroadway) [inline hippiestation/code/modules/mob/living/say.dm]: At this point I feel like we should setup some config file and have the code check through everything in said config for dumbo words.

**Comment by karmaisblackandbluepilled** [comment]
+1 i am racist against furries

**Review by ghost — DISMISSED** [review]

**Review by AnthonyJNgo — DISMISSED** [review]
Added nya and hola to this

**Review by AnthonyJNgo — APPROVED** [review]

**Pull request merged**